### PR TITLE
STIJ-286: Added tag-list component

### PR DESCRIPTION
**Pull request by delrueba**
## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/31-molecules/teaser/teaser--teaser--wide.twig [hunk 1]
```diff
@@ -1,19 +1,20 @@
+{% if tag %}
+  {% set myTags = [{text: tag, type: 'span-tag'}] %}
+{% endif %}
+{% if tag2 %}
+  {% set myTags = myTags|merge([{text: tag2, type: 'span-tag'}]) %}
+{% endif %}
```
Comment (jeroendegloire) [inline components/31-molecules/teaser/teaser--teaser--wide.twig]: Heeft tag2 een functie? Zoja welke componenten geven deze value door?

File: components/31-molecules/teaser/teaser--teaser--wide.twig [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-        {% include '@tag' with {
-          "tag_text": tag,
-          "type": 'span-tag'
-        } %}
-      {% endif %}
-      {% if tag2 %}
-        {% include '@tag' with {
-          "tag_text": tag2,
-          "type": 'span-tag'
+      {% if myTags %}
```
Comment (jeroendegloire) [inline components/31-molecules/teaser/teaser--teaser--wide.twig]: suggestie andere variablename voor 'myTags': gwn tags of tagList?

File: components/31-molecules/teaser/teaser--teaser--wide.twig (hunk 1 above)
Comment (delrueba) [reply components/31-molecules/teaser/teaser--teaser--wide.twig]: Is legacy, would have preferred an array myself. 
But I was lazy and didn't want to rewrite other components.

Comment (delrueba) [reply components/31-molecules/teaser/teaser--teaser--wide.twig]: Refactored tag & tag2 to tags array.

**Review by jeroendegloire — APPROVED** [review]

**Pull request merged**